test/integration/ggrc/converters/test_import_requests.py

```
@@ -23,6 +24,9 @@ class TestRequestImport(converters.TestCase):
 
   def setUp(self):
     """ Set up for Request test cases """
+    # These tests will be removed during next step:
```

These tests should not be removed, they should be rewritten to work with Assessments. So please change the comment.

The reason for this is, assessments and requests use the same handlers on the backend and it was easier to create mock data for requests, But these tests also catch errors with assessments, so it would be bad if we removed them.